### ISSUE TITLE
HTTPS Everywhere breaking Hangouts on Chrome

### ISSUE THREAD

**Issue by MentalEngineer** [EFForg/https-everywhere#14509]
[//]: # (Thank you for reporting an issue to HTTPS Everywhere.)
[//]: # (We welcome input from users on improving this project.)
[//]: #
[//]: # (Please help us by following this issue template.)
[//]: # (You can delete all blank lines and all lines starting)
[//]: # (with the comment marker, such as this one.)
[//]: #
[//]: # (Delete all but one of the following "Type" lines.)
[//]: # (Leave in the line that best describes the issue you are reporting.)

Type: other

[//]: # (If you are reporting a ruleset/website problem, include the top-level)
[//]: # (domain below. For example, if you want to report an issue about)
[//]: # ("one.example.com" and "two.example.com", then the line below should)
[//]: # (be:)
[//]: #
[//]: # (Domain: example.com)
[//]: #
[//]: # (Be sure to remove the parenthesis and comment marker.  If you are only)
[//]: # (reporting an issue about "one.example.com", then the line below should)
[//]: # (be:)
[//]: #
[//]: # (Domain: one.example.com) 
[//]: #
[//]: # (Only include one top-level domain. If you have more than one top-level)
[//]: # (domain to report, such as both "example.com" and "example.org", open a)
[//]: # (new issue for each top-level domain.)
[//]: #
[//]: # (If you are not reporting a ruleset/website problem, you can delete the)
[//]: # ("Domain" line or leave it blank.)
[//]: # (Include any other relevant information below. Thank you again for)
[//]: # (helping to improve HTTPS Everywhere.)

On Chrome 64.0.3282.119 with extension version 2018.1.29, HTTPS Everywhere prevents Hangouts from making or receiving calls and changing the speaker and microphone settings. I've been having the issue for at least a month. I imagine it's a problem with Chrome and not y'all, but I figured I'd mention it here just in case.

**Comment by wonderchook** [comment]
I tried to reproduce with Chrome 64.0.3282.119 with extension version 2018.1.29 and I was unable to reproduce.

**Comment by wonderchook** [comment]
Are you running any other plugins that might be interfering? What OS are you running?

**Comment by MentalEngineer** [comment]
Windows 10 version 1709. I disabled all plugins, which fixed the problem, and then re-enabled them one by one until it came back. It could be an interaction with another plugin that I'd already re-enabled, I guess - I'll try with only HTTPS Everywhere on and see what it does.

**Comment by jeremyn** [comment]
@MentalEngineer HTTPS Everywhere should not break Google Hangouts. This is a total guess, but Google Hangouts uses the WebRTC technology, which is optionally blocked by the uBlock Origin extension. Maybe you are blocking WebRTC with uBlock Origin?

**Comment by MentalEngineer** [comment]
That's what I thought it would be too! But uBlock Origin on/HTTPS Everywhere off, no problem. Both on, doesn't work.

**Comment by jeremyn** [comment]
@MentalEngineer I'm not using the exact same configuration as you are but I can't reproduce this problem either.

**Comment by wonderchook** [comment]
I tried installing uBlock Origin with both on I'm not able to reproduce the problem. I'm running Windows version 10.0.16299

**Comment by pipboy96** [comment]
Hangouts fully works with current version of HTTPS Everywhere and Chrome.
